src/VisualStudio/Core/Def/TaskList/HostDiagnosticUpdateSource.cs

```diff
@@ -127,5 +127,10 @@ public void ClearDiagnosticsForProject(ProjectId projectId, object key)
                 RaiseDiagnosticsRemovedForProject(projectId, key);
             }
         }
+
+        public DiagnosticData CreateAnalyzerLoadFailureDiagnostic(AnalyzerLoadFailureEventArgs e, string fullPath, ProjectId projectId, string language)
```

Comment:
This member exists on this interface just to deal with the fact CreateAnalyzerLoadFailureContext lives at the features layer and I can't get to it from the workspaces layer. I'm chatting with @mavasani to understand this better, since oddly we have two implementations of this, one of which is internal to the compiler, and we should just share that one.